<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgVisorWeb</class>
 <widget class="QDialog" name="dlgVisorWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visor de Paginas Web</string>
  </property>
  <widget class="QLabel" name="lblUrl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtURL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerWeb">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver Web</string>
   </property>
  </widget>
  <widget class="QWebEngineView" name="webVisor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1281</width>
     <height>931</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
